cmd/client/main.go

```diff
@@ -119,18 +121,20 @@ func withArgs(args []string, outf outFn, errorf errorFn) {
 	rootCmd.AddCommand(reportCmd(rootArgs, outf, errorf))
 	rootCmd.AddCommand(quotaCmd(rootArgs, outf, errorf))
 
-	if err := rootCmd.Execute(); err != nil {
-		errorf(err.Error())
-	}
+	return rootCmd
 }
 
 func main() {
-	withArgs(os.Args[1:],
+	rootCmd := GetRootCmd(os.Args[1:],
 		func(format string, a ...interface{}) {
 			fmt.Printf(format, a...)
 		},
 		func(format string, a ...interface{}) {
-			fmt.Fprintf(os.Stderr, format, a...)
-			os.Exit(1)
+			fmt.Fprintf(os.Stderr, format+"\n", a...)
+			os.Exit(-1)
```

Comment:
I want this to look like a command-line tool when you get the normal output, as opposed to seeing log-like output with timestamps and other metadata. That's the main motivation for using normal print calls into of log calls throughout the commands.